ocs_ci/helpers/odf_cli.py

```
@@ -166,6 +167,24 @@ def _set_executable_permissions(self):
+        """
+        Create an 'fdf' symlink pointing to the 'odf' binary for FDF clusters.
+
+        On non-FDF clusters this is a no-op.
+
+        """
+        if not config.DEPLOYMENT.get("fdf_cluster"):
+            return
+
+        fdf_path = os.path.join(os.path.dirname(self.local_cli_path), "fdf")
+        if os.path.exists(fdf_path):
+            log.info(f"FDF alias already exists at {fdf_path}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)